web/rest/brand/features/my/activeCalls/getActiveCalls.feature

```diff
@@ -11,9 +11,9 @@ Feature: Retrieve active calls
       And the JSON should be equal to:
     """
       {
-          "inbound": -1,
-          "outbound": -1,
-          "total": -2
+          "inbound": 1,
```

Comment:
I don't know why this has changed :man_shrugging:

Reply:
I added scan method to FakeRedisMasterFactory in https://github.com/irontec/ivoz-core/commit/52c2d8143c212336922e2b250f413f690495a82d to make the API tests work. This change was taken from the 3.x branch where this enhancement belongs.

I don't understand where the numbers come from, neither here  or in artemis.